### DISCUSSION TITLE
Problem with streams from Stripchat

### DISCUSSION THREAD

**Discussion by darkdiamond93** [aclap-dev/video-downloadhelper#278] [Help and Q&A]
### Your question

Since yesterday I'm having problems downloading streams from Stripchat: the download always stops after a few seconds. 
No problems with other websites.
OS used: MX Linux and Windows 10
Browsers: Firefox, Chrome, Chromium, Edge
VDH version: 8.2.1.1 stable
VdhCoApp: 2.0.19

Is there any workaround? 
Thanks in advance for any help.

### Addon & CoApp version and details

VDH8.2.1.1 VdhCoApp2.0.19

**Comment by GrampaWildWilly** [comment]
Let's have at least one URL of a web page where you have found a video that won't download.

**Reply by darkdiamond93** [reply]
It doesn't work with any model on the site.
For example, I just tried with this one and it doesn't work: it only downloads the first few seconds and then stops:
https://fr.stripchat.com/MoranaCarter

**Reply by GrampaWildWilly** [reply]
I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3235, Firefox 123.0.1 64-bit, licensed VDH 8.2.2.6a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user, like you.

You're right.

I tried to get relevant information out of the Network Monitor as well.  It was rather a nuisance to do because the web site is continuously sending stuff into the Network Monitor, even after I reloaded the page & did NOT click the play button on the video.  I filtered on "master" & did find a number of what appeared to be HLS master manifests.  This number gradually grew.  But my attempts to use any of them to download directly with ffmpeg resulted in files with durations of less than 10 seconds.  Somebody much smarter than myself will have to look into this.

**Reply by darkdiamond93** [reply]
Thanks for your confirmation. Everything was working until Friday, now I think they are using some sort of protection to avoid stream downloads. I'd want to download freely at least the FREE public shows of their models, and then I'll pay them if I want to watch their private shows!
I would be very grateful if anyone here could help to fix this problem...

**Comment by darkdiamond93** [comment]
Update: I tried in another PC with VDH8.2.1.1 + VdhCoApp2.0.10 and it downloads flawlessy, so I think the problem is with version 2.0.19 of the VdhCoapp. I suggest not to update to that version until this problem is resolved.

**Reply by Corvorum** [reply]
Thanks for the information. Finaly I could fix the problem by downgrading to VdhCoApp2.0.10.

**Reply by ethanhunt1459** [reply]
the downloaded video gets stuck for like 2-4 secs in an interval of 7-9 secs is there a fix for this?
